packages/cozy-mespapiers-lib/src/helpers/queries.js

```diff
@@ -51,9 +52,13 @@ export const buildFilesQueryWithQualificationLabel = () => {
         .partialIndex({
           'metadata.qualification.label': {
             $exists: true
-          }
+          },
+          'cozyMetadata.uploadedBy': { $exists: true }
         })
-        .indexFields(['metadata.qualification.label'])
+        .indexFields([
+          'metadata.qualification.label',
+          'cozyMetadata.uploadedBy'
```

Comment:
There is no need to index fields inside a partial index, as they actually won't be in the index itself